src/pages/Inventory.tsx

```diff
@@ -0,0 +1,213 @@
+import React, { useState } from "react";
+import "../stylesheets/Inventory.css";
+
+// used for temp unique item ids
+const generateRandomId = () => {
+  return Math.floor(Math.random() * 1000000);
+};
+
+export default function InventoryManagement() {
+  const [items, setItems] = useState<Item[]>([]);
+  const [selectedItem, setSelectedItem] = useState<Item | null>(null);
+  const [itemName, setItemName] = useState<string>("");
+  const [itemAbout, setItemAbout] = useState<string>("");
+  const [itemPrice, setItemPrice] = useState<number>(0.0);
+  const [itemQuantity, setItemQuantity] = useState<number>(0);
+  const [itemId, setItemId] = useState<number | null>(null);
+  
+
+  const handleInputNameChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setItemName(event.target.value);
+  };
+
+  const handleInputAboutChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setItemAbout(event.target.value);
+  };
+
+  const handleAddItem = async () => {
+    if (itemName.trim() !== "") {
+      try {
+        const newItem = {
+          name: itemName,
+          id: generateRandomId(),
+          about: itemAbout,
+          price: itemPrice,
+          quantity: itemQuantity,
+          businessId: 10, // Set BusinessId to 10, just something random I picked till we get that part set up more
+        };
+  
+        const response = await fetch('https://localhost:7079/api/inventory/add', {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify(newItem),
+        });
+  
+        if (response.ok) {
+          const createdItem = await response.json();
+          setItems(prevItems => [...prevItems, createdItem]);
+          setItemName("");
+          setItemAbout("");
+          setItemPrice(0);
+          setItemQuantity(0);
+        } else {
+          console.error('Failed to add item:', response.statusText);
+        }
+      } catch (error) {
+        console.error('Error adding item:', error);
+      }
+    }
+  };
+  
+
+  const handleItemClick = (item: Item) => {
+    setSelectedItem(item);
+    setItemId(item.id);
+    setItemName(item.name);
+    setItemAbout(item.about);
+    setItemPrice(item.price);
+    setItemQuantity(item.quantity)
+  };
+
+  const handleSaveItem = async () => {
+    if (selectedItem) {
+      try {
+        const updatedItem = {
+          id: selectedItem.id,
+          price: itemPrice,
+          quantity: itemQuantity,
+        };
+
+        const response = await fetch(`https://localhost:7079/api/inventory/update/`, {
```

Comment:
I think we should consider changing this to be a const variable somewhere.

As in,
`const inventoryApiPath = "https://localhost:7079/api/inventory";`

then using it in code like
`const response = await fetch('${inventoryApiPath}/update');`